deps/graph-parser/src/logseq/graph_parser/block.cljs

```diff
@@ -592,6 +592,22 @@
                 (assoc :block/updated-at updated-at))]
     (dissoc block :title :body :anchor)))
 
+(defn fix-duplicate-id
+  [block]
+  (-> block
+      (assoc :uuid (d/squuid))
+      (update :properties dissoc :id)
+      (update :properties-text-values dissoc :id)
+      (update :properties-order #(vec (remove #{:id} %)))
+      (update :content (fn [c]
+                         (let [replace-str (re-pattern
+                                            (str
+                                             "\n*\\s*"
+                                             (if (= :markdown (:format block))
+                                               (str "id:: " (:uuid block))
```

Comment:
Would be nice to use gp-property/colons* vars here so it's easy to identify and improve syntax later